pyrsh.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+
+def Main():
+
+	# Описание скрипта
+	parser = argparse.ArgumentParser(description='Remote command execution via ssh')
+	#  Задаем названия и описания аргументов
+	parser.add_argument('host', help='Hostname', type=str)
+	parser.add_argument('user', help='Your user', type=str)
+	parser.add_argument('password', help='Your password', type=str)
+	parser.add_argument('command', help='Your command', type=str)
```

Comment:
@GitSergeyGit the suggestion is to move parser definition to outer scope. `pyrsh` is considered to be cmdline tool, so it is fine to have `parser` defined globally(meaning it is not a library)
Lets also pass to the `main` already parsed arguments, agree?